gulp/tasks/build.js

```diff
@@ -172,13 +166,13 @@ function compileIndvES2015ModulesToBrowser() {
         },
         footer: fileName => {
           return isFirebaseApp(fileName) ? `
-          })();` : `
+          })().default;` : `
```

Comment:
I added a comment pointing to that url for reference. I finally opted not to use `babel-plugin-add-module-exports` since it needs CommonJS modules to do its thing, but webpack needs to work with ES2015 modules in order to perform tree shaking.